cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -597,9 +597,6 @@
     public static final String RETENTION_SECONDS = "metrics.data.table.retention.resolution";
 
     public static final String TOPIC_PREFIX = "metrics.topic.prefix";
-    public static final String KAFKA_TOPIC_PREFIX = "metrics.kafka.topic.prefix";
-    public static final String KAFKA_PARTITION_SIZE = "metrics.kafka.partition.size";
-    public static final String KAFKA_CONSUMER_PERSIST_THRESHOLD = "metrics.kafka.consumer.persist.threshold";
     public static final String KAFKA_META_TABLE = "metrics.kafka.meta.table";
```

Comment:
better to add a comment to clarify that this is a legacy name and Kafka is actually not used any more. The table stores meta data for TMS metrics topics